weblogic-azure-vm/arm-oraclelinux-wls-admin/src/main/arm/nestedtemplates/adminTemplateForCustomSSL.json

```diff
@@ -489,6 +495,20 @@
             }
          }
       },
+      {
+         "apiVersion": "${azure.apiVersionForPublicIPAddresses}",
+         "type": "Microsoft.Network/publicIPAddresses",
+         "tags": "[if(empty(parameters('tagsByResource')['${identifier.publicIPAddresses}']),createObject(parameters('const_guidTag'),''),union(parameters('tagsByResource')['${identifier.publicIPAddresses}'],createObject(parameters('const_guidTag'),'')))]",
+         "condition": "[equals(parameters('virtualNetworkNewOrExisting'), 'existing')]",
+         "name": "[variables('name_publicIPAddress')]",
```

Comment:
```
         "condition": "[equals(parameters('virtualNetworkNewOrExisting'), 'new')]",
         "name": "[variables('name_publicIPAddress')]",
```
You use the same name for ip address for both `new` and `existing` vNet:
```
         "condition": "[equals(parameters('virtualNetworkNewOrExisting'), 'existing')]",
         "name": "[variables('name_publicIPAddress')]",
```

Pls merge them into one resource with different values for properties impacted by value `parameters('virtualNetworkNewOrExisting')`.